docs/resources/dli_datasource_auth.md

```diff
@@ -0,0 +1,153 @@
+---
+subcategory: "Data Lake Insight (DLI)"
+---
+
+# huaweicloud_dli_datasource_auth
+
+Manages a DLI datasource authentication resource within HuaweiCloud.  
+
+## Example Usage
+
+### Create a datasource Password authentication
+
+```hcl
+  variable "username" {}
+  variable "password" {}
+  
+  resource "huaweicloud_dli_datasource_auth" "test" {
+    type     = "passwd"
+    name     = "demo"
+    username = var.username
+    password = var.password
+  }
+```
+
+### Create a datasource CSS authentication
+
+```hcl
+  variable "username" {}
+  variable "password" {}
+  variable "certificate_location" {}
+  
+  resource "huaweicloud_dli_datasource_auth" "test" {
+    type                 = "CSS"
+    name                 = "demo"
+    username             = var.username
+    password             = var.password
+    certificate_location = var.certificate_location
+  }
+```
+
+### Create a datasource Kafka_SSL authentication
+
+```hcl
+  variable "truststore_location" {}
+  variable "truststore_password" {}
+  variable "keystore_location" {}
+  variable "keystore_password" {}
+  variable "key_password" {}
+  
+  resource "huaweicloud_dli_datasource_auth" "test" {
+    type                = "Kafka_SSL"
+    name                = "demo"
+    truststore_location = var.truststore_location
+    truststore_password = var.truststore_password
+    keystore_location   = var.keystore_location
+    keystore_password   = var.keystore_password
+    key_password        = var.key_password
+  }
+```
+
+### Create a datasource Kerberos authentication
+
+```hcl
+  variable "username" {}
+  variable "krb5_conf" {}
+  variable "keytab" {}
+  
+  resource "huaweicloud_dli_datasource_auth" "test" {
+    type      = "KRB"
+    name      = "demo"
+    username  = var.username
+    krb5_conf = var.krb5_conf
+    keytab    = var.keytab
+  }
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String) The name of a datasource authentication.
+
+* `type` - (Required, String, ForceNew) Data source type.  
+  The options are as follows:
+    + **passwd**: Password.
+    + **CSS**: CSS.
+    + **KRB**: Kafka_SSL.
+    + **Kafka_SSL**: Kafka_SSL.
+
+  Changing this parameter will create a new resource.
+
+* `username` - (Optional, String) Username.
+
+* `password` - (Optional, String) Password.
```

Comment:
please add more information for the two parameters.

Reply:
done